scipy/optimize/_root.py

```diff
@@ -163,6 +164,7 @@ def root(fun, x0, args=(), method='hybr', jac=None, tol=None, callback=None,
 
     The solution can be found using the ``method='krylov'`` solver:
 
+    >>> import numpy as np
```

Comment:
This should be the top of the example section.

Reply:
I was thinking since this was a separate example, I should add `import numpy as np`. I'll remove it since it has already been added to the top of the examples section.